### PR TITLE
BREAKING CHANGE(web-twig): Remove `onClick` prop #DS-686

### PR DESCRIPTION
**Pull request by crishpeen**
## Description

Remove the deprecated `onClick` prop. Native `onclick` is available. 

### Additional context

Only Twig related. 

### Issue reference

#DS-686

---

### Before submitting the PR, please make sure you do the following

- [ ] Read the [Contributing Guidelines](https://github.com/lmc-eu/spirit-desing-system/blob/main/CONTRIBUTING.md).
- [ ] Follow the [PR Title/Commit Message Convention](https://github.com/lmc-eu/spirit-desing-system/blob/main/CONTRIBUTING.md#commit-conventions).
- [ ] Provide a description in this PR that addresses **what** the PR is solving, or reference the issue that it solves (e.g. `fixes #123`).
- [ ] Ideally, include relevant tests that fail without this PR but pass with it.

### CODE REVIEW

File: packages/web-twig/src/Resources/components/Button/stories/ButtonColorsSizes.twig [hunk 1]
```diff
@@ -8,7 +8,7 @@
       {% for color in colors %}
       <div class="{% if color is same as('inverted') %}docs-Box {% endif %}my-600">
         <div>
-          <Button color={color} size={size} onClick="alert('rimmer')">Button {{ color }}</Button>
+          <Button color={color} size={size} onclick="alert('rimmer')">Button {{ color }}</Button>
```
Comment (literat) [inline packages/web-twig/src/Resources/components/Button/stories/ButtonColorsSizes.twig:11]: Maybe we should use better alert then `rimmer`...

**Review by literat — APPROVED** [review]
Just a nitpicking about one alert.

File: packages/web-twig/src/Resources/components/Button/stories/ButtonColorsSizes.twig (hunk 1 above)
Comment (crishpeen) [reply packages/web-twig/src/Resources/components/Button/stories/ButtonColorsSizes.twig]: I think I know who is the author :D anyway fixed

Comment (literat) [reply packages/web-twig/src/Resources/components/Button/stories/ButtonColorsSizes.twig]: Thank you, you are the best 👍

**Pull request merged**